src/components/map/SelfCircle.tsx

```
@@ -1,26 +1,24 @@
-import { useContext, useEffect, useMemo, useRef, useState } from "react";
+import { useContext, useEffect, useState, useRef } from "react";
 import AppContext from "../../context/AppContext";
 import { checkPosition } from "../../utils";
-import { Circle, Marker, MarkerProps } from "react-leaflet";
-import L from "leaflet";
+import { Map, Marker } from "maplibre-gl";
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)